fusio/src/lib.rs

```diff
@@ -5,6 +5,7 @@ mod error;
 #[cfg(feature = "fs")]
 pub mod fs;
 pub mod local;
+pub mod parse;
```

Comment:
should add `#[cfg(feature = "fs")]`

Reply:
parse is not a clear mod name, parse what? place it to `src/fs/options.rs` might be a better choice.